src/sidebar/helpers/visible-threads.ts

```diff
@@ -11,10 +11,15 @@ export const THREAD_DIMENSION_DEFAULTS = {
   marginBelow: 800,
 };
 
-type VisibleThreads = {
+export type VisibleThreads = {
   visibleThreads: Thread[];
   offscreenUpperHeight: number;
   offscreenLowerHeight: number;
+
+  /** Number of hidden threads above the first visible one */
+  hiddenThreadsAbove: number;
+  /** Number of hidden threads below the last visible one */
+  hiddenThreadsBelow: number;
```

Comment:
`hiddenThreadsBelow` is calculated but not actually used. It is very cheap to calculate, so performance is not a concern, but I would probably remove it until we have a concrete need.

Reply:
Yeah, you are probably right.